### PR TITLE
Remove IS_BURN and the unused code behind the flag

### PR DESCRIPTION
**Pull request by goran-peoski-work**
No environments so far have set the flag so removing the final vestiges of the `IS_BURN` and the code behind it.

### CODE REVIEW

File: src/components/molecules/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -50,14 +47,6 @@ const TicketsPopover: React.FC<{ futureUpcoming: UpcomingEvent[] }> = (
   </Popover>
 );
 
-const GiftPopover = (
-  <Popover id="gift-popover">
-    <Popover.Content>
-      <GiftTicketModal />
```
Comment (sunny-viktoryia) [inline src/components/molecules/NavBar/NavBar.tsx:56]: i guess you can remove GiftTicketModal as well if it's not used anywhere.

File: src/pages/Admin/Venue/DetailsForm.tsx [hunk 2]
```diff
@@ -61,8 +52,6 @@ import { useShowHide } from "hooks/useShowHide";
 import { useSovereignVenue } from "hooks/useSovereignVenue";
 import { useUser } from "hooks/useUser";
 
-import { PlayaContainer } from "pages/Account/Venue/VenueMapEdition";
```
Comment (sunny-viktoryia) [inline src/pages/Admin/Venue/DetailsForm.tsx:64]: i guess you can remove PlayaContainer too if it's not used anywhere.

File: src/secrets.ts [hunk 3]
```diff
@@ -9,8 +9,8 @@ export const LOGROCKET_APP_ID = process.env.REACT_APP_LOGROCKET_APP_ID;
 export const MIXPANEL_PROJECT_TOKEN =
   process.env.REACT_APP_MIXPANEL_PROJECT_TOKEN;
 
+// @debt WS_RELAY_URL is not used in code other the next one line
 export const WS_RELAY_URL = process.env.REACT_APP_WS_RELAY_URL;
```
Comment (sunny-viktoryia) [inline src/secrets.ts:13]: if so, can be removed too, I believe.

File: src/settings.ts [hunk 4]
```diff
@@ -35,17 +34,11 @@ export const SPARKLEVERSE_PRIVACY_POLICY =
   "https://sparklever.se/privacy-policy/";
 export const PLATFORM_BRAND_NAME = "Sparkle";
 
-export const HOMEPAGE_URL = IS_BURN
-  ? SPARKLEVERSE_HOMEPAGE_URL
-  : SPARKLE_HOMEPAGE_URL;
+export const HOMEPAGE_URL = SPARKLE_HOMEPAGE_URL;
 
-export const TERMS_AND_CONDITIONS_URL = IS_BURN
-  ? SPARKLEVERSE_TERMS_AND_CONDITIONS_URL
```
Comment (sunny-viktoryia) [inline src/settings.ts:43]: this const SPARKLEVERSE_TERMS_AND_CONDITIONS_URL can be also removed.

File: src/settings.ts [hunk 5]
```diff
@@ -35,17 +34,11 @@ export const SPARKLEVERSE_PRIVACY_POLICY =
...
-  : SPARKLE_TERMS_AND_CONDITIONS_URL;
+export const TERMS_AND_CONDITIONS_URL = SPARKLE_TERMS_AND_CONDITIONS_URL;
 
-export const PRIVACY_POLICY = IS_BURN
-  ? SPARKLEVERSE_PRIVACY_POLICY
```
Comment (sunny-viktoryia) [inline src/settings.ts:47]: this const SPARKLEVERSE_PRIVACY_POLICY can be also removed.

**Review by sunny-viktoryia — COMMENTED** [review]
a bit more removal suggestions in the comments.

File: src/components/molecules/NavBar/NavBar.tsx (hunk 1 above)
Comment (goran-peoski-work) [reply src/components/molecules/NavBar/NavBar.tsx:56]: This should've been removed... maybe a commit was left unpushed.

Comment (goran-peoski-work) [reply src/components/molecules/NavBar/NavBar.tsx:56]: Ah, OK, the component. Will do.

Comment (goran-peoski-work) [reply src/components/molecules/NavBar/NavBar.tsx:56]: removed @ 3a6c225599966a5c1851a3acdd3fed0b15bda7c8

File: src/pages/Admin/Venue/DetailsForm.tsx (hunk 2 above)
Comment (goran-peoski-work) [reply src/pages/Admin/Venue/DetailsForm.tsx:64]: removed @ c9e363a5ad5f58002856101536d76b824738f161

File: src/secrets.ts (hunk 3 above)
Comment (goran-peoski-work) [reply src/secrets.ts]: removed @ 5a0e0729d2cdb8571b4f3c7bbc88cbedc7195eac

File: src/settings.ts (hunk 4 above)
Comment (goran-peoski-work) [reply src/settings.ts:43]: removed @ dc24c4cb9d72c54adeb66ae2ff1beeb50ebe8beb

File: src/settings.ts (hunk 5 above)
Comment (goran-peoski-work) [reply src/settings.ts:47]: removed @ dc24c4cb9d72c54adeb66ae2ff1beeb50ebe8beb

**Review by goran-peoski-work — COMMENTED** [review]
Removed extra dangling `IS_BURN` dependencies

**Review by yurii-lubynets — COMMENTED** [review]
Looks clear for me

**Review by sunny-viktoryia — APPROVED** [review]
Looks good to me. Thanks for the refactoring. 🤗

**Pull request merged**